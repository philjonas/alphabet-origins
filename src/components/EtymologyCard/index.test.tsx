import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() })
import {EtymologyCard} from '.';


test('matches snapshot', () => {
  const wrapper = shallow(<EtymologyCard />)
  expect(toJson(wrapper)).toMatchSnapshot();
})