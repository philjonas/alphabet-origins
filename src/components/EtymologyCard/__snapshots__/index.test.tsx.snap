// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="logo-container"
>
  <div
    className="letter-container"
  >
    <LogogramBox
      isSmaller={true}
      legend="Egyptian"
      symbol="𓃾"
    />
    <Divider
      barrier="..."
    />
    <LogogramBox
      legend="Phoenician"
      symbol="𐤀"
    />
    <Divider
      barrier="..."
    />
    <LogogramBox
      legend="Greek"
      symbol="Αα"
    />
    <Divider
      barrier="..."
    />
    <LogogramBox
      legend="Latin"
      symbol="Aa"
    />
  </div>
  <h2
    className="letter-meaning"
  >
    Ox, head of cattle
  </h2>
</div>
`;
